src/types/User.ts

```diff
@@ -11,4 +11,6 @@ export type User = {
 export interface RequestWithUser extends Request {
     user?: User;
     isAuthenticated(): boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    query: any;
```

Comment:
This rolls the same way as the other review, unless absolutely impossible to do so, I'd like to see an avoidance in disabling tsconfig rules or changing them from strict mode.

```suggestion
    query?: string | 'other type it could be';
```